document-store/src/main/java/org/hypertrace/core/documentstore/Datastore.java

```diff
@@ -17,4 +17,6 @@ public interface Datastore {
 
   @SuppressWarnings("unused")
   DocStoreMetricProvider getDocStoreMetricProvider();
+
+  void close();
```

Comment:
Is this interface not exposed outside, right?

Reply:
Yes. Intended to expose this so that the clients can proactively close the connections at the service stop (if they need to) instead of relying on the DB to terminate the connections.